api/types/app.go

```diff
@@ -36,11 +35,15 @@ func ConvertToAppConfigValues(kotsConfigValues *kotsv1beta1.ConfigValues) AppCon
 
 	configValues := make(AppConfigValues)
 	for key, value := range kotsConfigValues.Spec.Values {
+		// password types will have ValuePlaintext set instead of Value. Let's not break compatibility for now
+		finalValue := value.Value
+		if value.Value == "" && value.ValuePlaintext != "" {
+			finalValue = value.ValuePlaintext
+		}
```

Comment:
just calling out that value will take precedence over valuePlaintext

Reply:
Yeah that's by design, `Value` is used by all the other "value based fields" afaik except for `password`.